CHANGELOG.md

```diff
@@ -1,17 +1,64 @@
-Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CONTRIBUTING.md) on how to contribute to Cucumber.
+Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CONTRIBUTING.md)
+on how to contribute to Cucumber.
 
-## [master](https://github.com/cucumber/cucumber-rails/compare/v2.0.0...master) (Not yet released)
+## [master](https://github.com/cucumber/cucumber-rails/compare/v2.1.0...master) (Not yet released)
 
 ### New Features
 
  * 
 
 ### Changed
 
- * 
+ *
 
 ### Fixed
 
+ *
+
+## [v2.1.0](https://github.com/cucumber/cucumber-rails/compare/v2.0.0...v2.1.0) (2020-06-15)
+
+### New Features
+
+ * Added metadata to allow gem to be searched / indexed by rubygems
+    ([#447](https://github.com/cucumber/cucumber-rails/pull/447) [orien])
+    
+ * Allow Cucumber 4 to be used as an option for cucumber-rails
```

Comment:
If the verb tense used throughout this document is to be... uniform, then _perhaps_ this verb could be `Allowed Cucumber...`? Not a sticking point for me, but I noticed there'd been edits to that effect in this PR, and wanted to let you know.

Reply:
My 2 cents: I have a feeling that in general it's easier to use present tense, but either style is fine by me, and even inconsistent style.